docs/learn/learn-inflation.md

```diff
@@ -35,46 +36,49 @@ network through funding the on-chain treasury. There is an opportunity cost of k
 {{ polkadot: DOT :polkadot }}{{ kusama: KSM :kusama }} tokens idle with the current inflation model
 as the tokens get diluted over time. Economics and game theory suggest that setting an ideal
 inflation rate is essential for incentivizing the network participants as well as the growth of the
-network, and any deviation from it can have negative effects. Reducing the inflation rate could
-limit growth while increasing the inflation rate could break the incentive model of the token.
-Hence, **token inflation rate is not a forever fixed value, and inflation can be updated in the
-future through [on-chain governance](./learn-polkadot-opengov.md)** based on thorough tokenomics
-research.
+network, and any deviation from it can have adverse effects. Reducing the inflation rate could limit
+growth, while increasing the inflation rate could break the incentive model of the token. Hence,
+**token inflation rate is not a forever fixed value, and inflation can be updated in the future
+through [on-chain governance](./learn-polkadot-opengov.md)** based on thorough tokenomics research.
 
 ## Inflation Model
 
 The chart below shows the inflation model of the network. Depending on the number of staked tokens,
 the distribution of the inflation to validators and nominators versus the treasury will change
 dynamically to provide incentives to participate (or not participate) in staking.
 
-There is a [dynamic _ideal staking rate_](#ideal-staking-rate) (in the figure set to 0.5 or 50%)
-that the network tries to maintain. The goal is to have the _system staking rate_ meet the _ideal
-staking rate_. The system staking rate would be the total amount staked over the total token supply,
+There is a [dynamic _ideal staking rate_](#ideal-staking-rate) (in the figure set to 0.6 or 60%)
+that the network tries to maintain. The inflation model will incentivize network participants to
+stake when the _system staking rate_ < _ideal staking rate_ and disincentivize staking when _system
```

Comment:
At first glance, the word "system staking rate" is ambiguous by a new reader, making a reader think it is somehow dependent on the system or protocol. An apt name would be "current staking rate" or "staking rate in the current era" etc. If we already agreed on standardizing "system staking rate" in earlier discussions that I', not aware of, you can ignore this suggestion.

```suggestion
stake when the _current staking rate_ < _ideal staking rate_ and disincentivize staking when _current
```